pkg/client/client.go

```diff
@@ -162,3 +179,42 @@ func (c *Client) GetClusterKubeConfig(ctx context.Context, clusterName string, c
 
 	return string(kc), nil
 }
+
+// GetHelmValues retrieves the helm values of a Helm release in the provided
+// name and namespace and it will Unmarshal the values into the provided values
+// struct.
+func (c *Client) GetHelmValues(name, namespace string, values interface{}) error {
+	helmClient, err := c.getHelmClient(namespace)
```

Comment:
It would be good to check that `values` is a valid pointer similar to how `json.Unmarshal` does.
```suggestion
func (c *Client) GetHelmValues(name, namespace string, values interface{}) error {
	rv := reflect.ValueOf(values)
	if rv.Kind() != reflect.Pointer || rv.IsNil() {
		return fmt.Errorf("Values must be a pointer, instead got %v", reflect.TypeOf(values))
	}

	helmClient, err := c.getHelmClient(namespace)
```